Add frontpage

hugopeixoto opened the pull request:
Some functionalities were removed:

- frontpage_extras no longer works, since it depends on spline plugins
- no support for user last_seen_time, since we don't have users anymore

Things that are missing:

- cron + cache the entries, it's being fetched on every request
- last_seen_time cookie is not being set

hugopeixoto committed:
Add frontpage as-is

hugopeixoto committed:
Port frontpage to pyramid

Some functionalities were removed:

- frontpage_extras no longer works, since it depends on spline plugins
- no support for user last_seen_time, since we don't have users anymore

Things that are missing:

- cron + cache the entries, it's being fetched on every request
- last_seen_time cookie is not being set

magical commented:
Thanks!

Couple comments: 
* FrontPageController should be turned into a view, same as the other pokedex pages
* Just add the frontpage code to `splinext.pokedex`, no need to keep it in a separate package
* You can look at https://github.com/magical/spline-pokedex/blob/pyramid_pokedex/splinext/pokedex/views/caching.py#L36 for an example of how to do caching

That said, if this is basically working then i'd be fine with merging it as-is and fixing it up later.

hugopeixoto committed:
Move frontpage module into splinext.pokedex

I've removed the controller concept and implemented frontpage as a view.
Renames the templates directory from front_page to frontpage.

Cookies are now working (do we need an European banner now?).

I had forgot to add CSS. That's now working as well.

hugopeixoto commented:
I've removed the controller concept and implemented frontpage as a view.
Renames the templates directory from front_page to frontpage.

Cookies are now working (do we need an European banner now?).

I had forgot to add CSS. That's now working as well.
